api/v1beta1/disruption_types.go

```diff
@@ -138,19 +140,25 @@ func (s *DisruptionSpec) validateGlobalDisruptionScope() error {
 		return errors.New("cannot target specific containers because the level configuration is set to node")
 	}
 
+	if s.ContainerFailure != nil && s.Level == chaostypes.DisruptionLevelNode {
```

Comment:
Can you please add a `// Rule: ...` command on top of this condition just to keep it clear (even if the condition itself is clear enough).

Reply:
Sure, done